core/simulations/configs/surplusAuctionSimulation.ts

```diff
@@ -0,0 +1,21 @@
+import { warpTime, resetBlockchainFork } from '../../helpers/hardhat';
+import { HARDHAT_PRIVATE_KEY, HARDHAT_PUBLIC_KEY } from '../../helpers/constants';
+export default {
+    title: 'Fork block with active surplus auction',
+    steps: [
+        {
+            title: 'Reset blockchain fork',
+            // Block with various states of surplus auctions at 14078339,
+            entry: async () => {
+                console.info(
+                    `Wallet Credentials: public key: ${HARDHAT_PUBLIC_KEY}, private key: ${HARDHAT_PRIVATE_KEY}`
+                );
```

Comment:
As suggested, can we move this to the `resetBlockchainFork` and print it using `signerPrivateKey` variable and `provider.getAddress()`? (And don't forget to remove this logging from all other simulation configs)

In general, you should print information into the console where it is actually used, otherwise in case of the code change printed information will be invalid and misleading

Reply:
`provider` is `hre.network.provider` of `EtheriumProvider` type, it's not a signer. To print public key in this function as you suggest one would have to extract the signer first. See the change.